boards/arm/nrf52833_pca10111/nrf52833_pca10111_defconfig

```diff
@@ -0,0 +1,13 @@
+CONFIG_ARM=y
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52833_QIAA=y
+CONFIG_BOARD_NRF52833_PCA10111=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# additional board options
+CONFIG_GPIO_AS_PINRESET=y
```

Comment:
`CONFIG_NFCT_PINS_AS_GPIOS=y` should stay here, as P0.10 is used for `button_0`.

Reply:
ah... right

Reply:
added